src/isolate.rs

```diff
@@ -221,6 +256,10 @@ extern "C" fn pre_dispatch(
   control_buf: libdeno::deno_buf,
   data_buf: libdeno::deno_buf,
 ) {
+  // for metrics
+  let control_bytes_sent = control_buf.data_len as u64;
+  let data_bytes_sent = data_buf.data_len as u64;
```

Comment:
It would be good to update the control_bytes_sent and data_bytes_sent before the op is dispatched.
Async ops might take a long time to complete.
It would be good to have a test for this behavior too.

Reply:
I've added the logic for keeping track of them. Not sure how to test it with async operation though.